### ISSUE TITLE
Storybook: StringList component

### ISSUE THREAD

**Issue by torchiaf** [rancher/dashboard#7333]
Add the new component StringList to the Storybook.
relates to: rancher/dashboard#7326

**Comment by catherineluse** [comment]
This component already exists - for example, in the Service creation form, you can add a list of strings for external IPs:

<img width="950" alt="Screenshot 2022-11-05 at 2 44 59 PM" src="https://user-images.githubusercontent.com/20599230/200142459-f49600f2-1f0c-44f0-aa82-9575632133bd.png">

The component is in `shell/components/form/ArrayList`.

**Comment by torchiaf** [comment]
> The component is in `shell/components/form/ArrayList`.

Hi @catherineluse , thanks for the hint.
We talked about that component in the Slack channel but It seems more specific feature are required, like inline-edit or Remove  button unique for all elements a t the bottom.
Furthermore, the layout should be more suitable to a Desktop app (Rancher Desktop). At the end we decided to write a new component...
